app/src/main/res/layout/dialpad.xml

```diff
@@ -18,6 +18,7 @@
         android:layout_marginStart="@dimen/activity_margin"
         android:background="?attr/selectableItemBackgroundBorderless"
         android:contentDescription="@string/one"
+        android:visibility="visible"
```

Comment:
no need to set Visible everywhere, that is the default one

Reply:
I see, I will remove it.